test/ui/cypress/integration/serving.spec.js

```diff
@@ -187,6 +187,13 @@ describe('OCP UI for Serverless', () => {
   })
 
   it('can deploy a cluster-local service', () => {
+    const ocpVersion = Cypress.env('OCP_VERSION')
+    cy.semver(ocpVersion).then((semver) => {
+      // TODO: set proper 4.6.x version when BZ gets targeted in advisory:
+      //       https://bugzilla.redhat.com/show_bug.cgi?id=1978159
+      const range = '>=4.8 || ~4.7.18 || ~4.6.39'
```

Comment:
The openshift/console#9389 needs to be assigned to future OCP version (https://bugzilla.redhat.com/show_bug.cgi?id=1978159)